### ISSUE TITLE
Problem: Latest Bionic SS package was created and published with version 1.8.0 instead of 0.13.0

### ISSUE THREAD

**Issue by mamedin** [archivematica/Issues#397]
**Expected behaviour**

The latest bionic SS package version is 0.13.0

**Current behaviour**

The latest bionic package version is 1.8.0-1 of 0.13.0.-1 version.

This package was created by error and published in the official repository.

The storage-service bionic am-packbuild changelog has been updated with this version too, so new packages are created with 1.8.0 because it is higher than 0.13.0:

```
Setting up archivematica-storage-service-build-deps (1:1.8.0-1~18.04) ...
++ echo 8863492193a8f07e89fa6f1a27ad9889be4de822
+ dch -v 1:0.13.0-1~18.04 commit: 8863492193a8f07e89fa6f1a27ad9889be4de822
dch: fatal error at line 1092:
New version specified (1:0.13.0-1~18.04) is less than
the current version number (1:1.8.0-1~18.04)!  Use -b to force.
```

**Steps to reproduce**

Listing official bionic repository packages.

**Your environment (version of Archivematica, OS version, etc)**

All

---
**For Artefactual use:**
Please make sure these steps are taken before moving this issue from Review to Verified in Waffle:

- All PRs related to this issue are properly linked 👍
- All PRs related to this issue have been merged 👍
- Test plan for this issue has been implemented and passed 👍
- Documentation regarding this issue has been written and it has been added to the release notes, if needed 👍

**Comment by mamedin** [comment]
The `archivematica-storage-service` 1.8.0-1 version package was removed from the Bionic 1.8.0 repository. The latest version available at the moment is the expected 0.13.0-1 version.

When the `archivematica-storage-service-1.8.0-1` package has been installed, it has to be downgraded to the 0.13.0-1 version or newer. This way the package can be updated to future versions greater than 0.13.0-1 and lesser than 1.8.0-1. 

First update your apt repositories:

```
sudo apt-get update
```

Check which `archivematica-storage-service` package version was installed:

```
ubuntu@mamedin-bionic:~$ sudo dpkg -l | grep storage-service
ii  archivematica-storage-service        1:1.8.0-1~18.04                             amd64        Django webapp for managing storage in an Archivematica
```

List available packages:

```
ubuntu@mamedin-bionic:~$ sudo apt-cache policy archivematica-storage-service
archivematica-storage-service:
  Installed: 1:1.8.0-1~18.04
  Candidate: 1:1.8.0-1~18.04
  Version table:
 *** 1:1.8.0-1~18.04 100
        100 /var/lib/dpkg/status
     1:0.13.0-1~18.04 500
        500 http://packages.archivematica.org/1.8.x/ubuntu bionic/main amd64 Packages
```

Finally, please downgrade to` archivematica-storage-service-0.13.0-1` version, run:

```
sudo apt-get install archivematica-storage-service=1:0.13.0-1~18.04
```

**Comment by sromkey** [comment]
haha fixed now!!
